shared/src/main/scala/mlscript/Codegen.scala

```diff
@@ -250,7 +250,7 @@ object JSBinary {
       "instanceof" -> 12,
       "==" -> 11,
       "&&" -> 7,
-      "||" -> 7,
+      "||" -> 7
```

Comment:
Can we leave ending commas? I find them more convenient to edit and consistent. If you're using scalafmt: `trailingCommas =  always`.

Reply:
Sure.

Reply:
I set `trailingCommas` to `keep` in order to prevent introducing too many changes.

Reply:
Ok 👍